src/App.tsx

```diff
@@ -15,48 +16,147 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortFields {
+  byAlfabet = 'alphabet',
```

Comment:
```suggestion
enum SortBy {
  Alfabet = 'alphabet',
```
CamelCase for enum keys